JM/CF/Scripts/3_Game/CommunityFramework/Game/DayZGame.c

```diff
@@ -1,5 +1,7 @@
 modded class DayZGame
 {
+	ref CF_EventRPCArgs m_CF_EventRPCArgs = new CF_EventRPCArgs();
+
```

Comment:
Shouln't this also be autoptr?

Reply:
It works as-is with no reports of memory leaks. I think the problem may have been where passing new instances into a function. Never gets a chance for it's ref count to be updated when the function call ends.